packages/core/lib/queues/AbstractQueueService.ts

```diff
@@ -165,14 +165,27 @@ export abstract class AbstractQueueService<
   }
 
   protected logProcessedMessage(
-    _message: MessagePayloadSchemas | null,
+    message: MessagePayloadSchemas | null,
     processingResult: MessageProcessingResult,
     messageId?: string,
   ) {
+    const messageTimestamp = message ? this.tryToExtractTimestamp(message) : undefined
```

Comment:
I would do the check on the log level for this.logger first and skip it entirely if it's below debug

Reply:
From what I see Logger is a generic interface and it does not expose any property for log level. We could check `level` property as it's available in `pino`, but it want apply all of the time

Reply:
we should switch to CommonLogger from node-core, it exposes the isLevelEnabled method

Reply:
So we should probably change it everywhere and since logger is provided from outside that will be breaking change. Should I address it in separate PR then?

Reply:
yeah, makes sense